addons/point_of_sale/migrations/13.0.1.0.1/end-migration.py

```diff
@@ -5,7 +5,45 @@
 
 
 def create_pos_payment_methods(env):
-    env["pos.config"].post_install_pos_localisation()
+    """We will create a Payment method for each journal used on configurations"""
+    if not openupgrade.column_exists(
+            env.cr, "pos_payment_method", "old_journal_id"):
+        openupgrade.logged_query(
+            env.cr, """
+            ALTER TABLE pos_payment_method
+            ADD COLUMN old_journal_id int"""
+        )
+        journal_ids = [aj[0] for aj in env.cr.fetchall()]
```

Comment:
```suggestion
        journal_ids = [aj[0] for aj in env.cr.fetchall()]
```
falta query amb select, no?

Reply:
Si, l'he borrat sense voler, merci!!!